docs/customize/static_pages.md

```diff
@@ -60,30 +60,6 @@ Now that your custom page is configured, you can access it from the path you def
 
 ![Custom page no content](./img/custom-page_no-content.png)
 
-While adding or removing pages can only be done by the developer of an instance, editing a page can be done by an admin user, in the admin-panel at `/administration/pages`.
-
-Once configured, you can access the pages in the administration panel from the side menu.
-
-![Admin panel menu pages](./img/admin-panel-menu_pages.png)
-
-Here you will see a list of the pages you configured in the previous steps.
-
-![Admin panel pages](./img/admin-panel_pages.png)
-
-To edit one of your pages, you can click the edit button in the "Actions" column.
-
-This will take you to the edit-form of the page, which will look like this:
-
-![Admin panel pages edit](./img/admin-panel_pages_edit.png)
-
-As you can see, the title and description we added in the configuration step is already defined in the form. To add some content to the page, you can fill in the "Content" field and click save:
-
-![Admin panel pages edit](./img/admin-panel_pages_content.png)
-
-Now we can go to our custom page again and see the content displayed on the page.
-
-![Custom page with content](./img/custom-page_with-content.png)
-
```

Comment:
I relocated this section to the bottom because, if I'm not mistaken, the following section pertains to v11 to avoid flipping versions back and forth.

Reply:
Looks like the full "Edit pages" section is v12 specific right? We can move it all below and tag it as such. This way the heading is for the functionality (more navigable). We could also point out in the intro paragraph of this page that this is the case:

```
InvenioRDM supports an easy "out-of-the-box" way of creating static pages, basically web pages for which the HTML content is stored in the database.
And you can change them using the administration panel in v12.
```